packages/kas/package.json

```diff
@@ -22,7 +22,7 @@
         "dist"
     ],
     "scripts": {
-        "gen:parsers": "node src/parser-generator.js",
+        "gen:parsers": "node src/parser-generator.ts",
```

Comment:
Does this work (passing a TypeScript file to node directly)?

Previously we had to use SWC to run TS scripts as command line apps: https://github.com/Khan/perseus/blob/main/utils/pre-publish-check-ci.ts#L1

Reply:
It seems to work when I run it 🤷